src/server/cluster/cluster_family.cc

```diff
@@ -149,21 +149,24 @@ ClusterShardInfo ClusterFamily::GetEmulatedShardInfo(ConnectionContext* cntx) co
                                   ? static_cast<uint16_t>(absl::GetFlag(FLAGS_port))
                                   : cluster_announce_port;
 
-    info.master = {.id = id_, .ip = preferred_endpoint, .port = preferred_port};
+    info.master = {{.id = id_, .ip = preferred_endpoint, .port = preferred_port}, NodeHealth::NONE};
 
     if (cntx->conn()->IsPrivileged() || !absl::GetFlag(FLAGS_managed_service_info)) {
       for (const auto& replica : server_family_->GetDflyCmd()->GetReplicasRoleInfo()) {
-        info.replicas.push_back({.id = replica.id,
-                                 .ip = replica.address,
-                                 .port = static_cast<uint16_t>(replica.listening_port)});
+        info.replicas.push_back({{.id = replica.id,
+                                  .ip = replica.address,
+                                  .port = static_cast<uint16_t>(replica.listening_port)},
+                                 NodeHealth::NONE});
```

Comment:
to support this for cluster emulated mode we need here to check what is the status of the replica. i.e if its in loading state.
@romange do we want do this for emulated mode as well?